static/js/workbooks.js

```diff
@@ -1118,7 +1125,7 @@ require([
             setPlotPanelHeight(active_sheet);
         }
     });
-
+valid_plot_settings
     // validate the plot settings before initiating the plot
```

Comment:
This text ('valid_plot_settings') just seems to be floating?

Reply:
oopsie. please remove it.